cmd/copyUploadEnumerator.go

```diff
@@ -30,6 +30,11 @@ func (e *copyUploadEnumerator) enumerate(cca *cookedCopyCmdArgs) error {
 		return fmt.Errorf("cannot find source to upload")
 	}
 
+	// if the user specifies a virtual directory ex: /container_name/extra_path
+	// then we should extra_path as a prefix while uploading
```

Comment:
typo, missing verb

Reply:
I literally just copied the majority of a working test 👀, fixing the wording

Reply:
oh yikes